### DISCUSSION TITLE
Bingada crate is absent from Alire website

### DISCUSSION THREAD

**Discussion by Hubilari** [alire-project/alire#1411] [Support]
Hi,
trying to install bingada
dubois@<rep>$ alr get --build bingada
error: Crate [bingada] does not exist in the catalog.
$ 
without succes, but preesnt in Alre website.
Thanks.
Mark

**Comment by mosteo** [comment]
Possibly outdated community index, try `alr index --update-all` first.

**Reply by Hubilari** [reply]
This is OK now after refreshing the index.
What's the frequency refreshment of the index or the procedure, please?
Thanks

**Reply by mosteo** [reply]
The index at present is never updated automatically, you should do it whenever it feels necessary. But see https://github.com/alire-project/alire/issues/1348.
